ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/event-handlers.yml

```diff
@@ -15,3 +15,31 @@ event_handlers:
     trigger: on-boot
     action_type: bash
     action: echo "on-boot"
+  - name: trigger-on-maintenance
+    trigger: on-maintenance
+    trigger_on_maintenance:
+      operation: enter
+      interface: Ethernet4
+      stage: all
+    action_type: bash
+    action: echo "on-maintenance"
+  - name: trigger-on-maintenance1
+    trigger: on-maintenance
+    trigger_on_maintenance:
+      operation: enter
+      bgp:
+        neighbor_address_ipv4: 10.0.0.2
+      interface: Management3
```

Comment:
i dont see interface option for BGP peeer

Reply:
this is the case when bgp and interface both are defined, but it will take bgp only, as it is checked first in the template.

Reply:
Fixed